apis/securesourcemanager/v1alpha1/instance_reference.go

```diff
@@ -183,6 +184,91 @@ func (r *SecureSourceManagerInstanceRef) ResourceID() (string, error) {
 	return resourceID, nil
 }
 
+type PrivateCACAPoolRef struct {
```

Comment:
Suggest create a file privatecaref.go under apis/refs/v1beta1, similar to bigqueryref.go: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/apis/refs/v1beta1/bigqueryref.go

I'm going to refactor the resource references later, put them in one place will make the work easier.

Reply:
Done. The dev docs do mention this, so good point.

Reply:
Fixing..

Reply:
Done